.github/workflows/ci.yaml

```diff
@@ -0,0 +1,106 @@
+name: ci
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+jobs:
+  test:
+    name: test
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        toolchain: [stable]
+        build: [linux, macos, win]
+        include:
+        - build: linux
+          os: ubuntu-latest
+        - build: macos
+          os: macos-latest
+        - build: win
+          os: windows-latest
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Install Rust toolchain
+      uses: actions-rs/toolchain@v1
+      with:
+        toolchain: ${{ matrix.toolchain }}
+        profile: minimal
+        override: true
+
+    - name: Cache Rust dependencies 
+      uses: actions/cache@v2
+      with:
+        path: target
+        key: ${{ runner.os }}-cargo-test-${{ matrix.toolchain }}-${{ hashFiles('**/Cargo.lock') }}
+
+    - name: Build binary
+      run: cargo build --release
+      env:
+        CARGO_INCREMENTAL: 0
+        RUSTFLAGS: "-C debuginfo=0 -D warnings"
```

Comment:
What do these environment variables do?

Reply:
I've removed these. My idea was to reduce the disk space used on the runner but now when I think about it I doubt we will be using that much space.